### ISSUE TITLE
[Issue]: TypeError: bad operand type for unary -: 'NoneType'

### ISSUE THREAD

**Issue by gobshyte** [vladmandic/automatic#585]
### Issue Description

Hi
Had this running fine for about a day, but now whatever I try (different models, different settings, no vae, no extras enabled) I always get this TypeError when I click Generate.

```
INFO     Python 3.10.6 on Windows
INFO     Version: 1f6261be Thu Apr 27 13:12:20 2023 -0400
INFO     Latest published version: bfe62127e9d90015ba4c24df55333163bf53c73a 2023-04-27T19:47:13Z
INFO     Setting environment tuning
INFO     nVidia toolkit detected
INFO     Torch 2.0.0+cu118
INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
INFO     Torch detected GPU: NVIDIA GeForce RTX 3090 VRAM 24576 Arch (8, 6) Cores 82
INFO     Verifying requirements
INFO     Running setup
INFO     Installing packages
INFO     Installing repositories
INFO     Installing submodules
INFO     Updating submodules
INFO     Extensions enabled: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
         'multidiffusion-upscaler-for-automatic1111', 'prompt-bracket-checker', 'ScuNET',
         'sd-dynamic-thresholding', 'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation',
         'sd-extension-system-info', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel',
         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
INFO     Extensions enabled: ['sd-dynamic-prompts', 'stable-diffusion-webui-inspiration']
INFO     Updating Wiki
INFO     Running extension preloading
INFO     Server arguments: ['--listen', '--port', '7864']






gradio call: TypeError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\SDAI\automatic\modules\call_queue.py:58 in f                                                                     │
│                                                                                                                     │
│    57 │   │   │   │   pr.enable()                                                                                   │
│ >  58 │   │   │   res = list(func(*args, **kwargs))                                                                 │
│    59 │   │   │   if shared.cmd_opts.profile:                                                                       │
│                                                                                                                     │
│ C:\SDAI\automatic\modules\call_queue.py:38 in f                                                                     │
│                                                                                                                     │
│    37 │   │   │   try:                                                                                              │
│ >  38 │   │   │   │   res = func(*args, **kwargs)                                                                   │
│    39 │   │   │   finally:                                                                                          │
│                                                                                                                     │
│                                               ... 9 frames hidden ...                                               │
│                                                                                                                     │
│ C:\SDAI\automatic\modules\sd_hijack_clip.py:250 in process_tokens                                                   │
│                                                                                                                     │
│   249 │   │                                                                                                         │
│ > 250 │   │   z = self.encode_with_transformers(tokens)                                                             │
│   251                                                                                                               │
│                                                                                                                     │
│ C:\SDAI\automatic\modules\sd_hijack_clip.py:298 in encode_with_transformers                                         │
│                                                                                                                     │
│   297 │   def encode_with_transformers(self, tokens):                                                               │
│ > 298 │   │   outputs = self.wrapped.transformer(input_ids=tokens, output_hidden_states=-opts.                      │
│   299                                                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
TypeError: bad operand type for unary -: 'NoneType'
```

**Comment by byteconcepts** [comment]
I can confirm this error.
Get the same error from time to time but not frequently.

updated: 2023-04-27
hash: bfe62127
url: https://github.com/vladmandic/automatic/tree/master

arch: AMD64
cpu: Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
system: Windows
release: Windows-10-10.0.19044-SP0
python: 3.10.8

**Comment by vladmandic** [comment]
i cannot reproduce, but based on the traceback i can guess what is happening so i've made a blind change to handle the error.
try updating the repo and if issue persists, report here and i'll reopen it.

**Comment by gobshyte** [comment]
Excellent - yes it seems it was clip skip set to none - somehow. That was one of the settings I failed to tinker with trying to fix it.

Before I updated, I went back in and the error was still happening, so adjusted clip skip (it was saying it was 1, despite the error), and it then started to work, on both 2 and 1. (I even tried typing in zero but it wouldn't take it, haha)

Now updated and all still working :)

Many thanks for the awesome work and amazingly fast fix!!!
